js/speedgrader.js

```diff
@@ -0,0 +1,9 @@
+$(document).ready(function(){
+    if ($.browser.mozilla && window.location.href.indexOf('gradebook/speed_grader')) {
```

Comment:
Shouldn't that be `(window.location.href.indexOf('gradebook/speed_grader') > -1)`?